src/utils/utils.ts

```diff
@@ -141,3 +141,17 @@ export function colorFromId(id: number): string {
 
   return `${prefix}${color.toString().padStart(2, "0")}`;
 }
+
+/**
+ * Check if the passed CSS selector is valid.
+ * @param selector CSS selector to validate
+ */
+export function isSelectorValid(selector: string): boolean {
```

Comment:
what is this used for? it is not used anywhere

Reply:
I started developing this feature with the idea of a `current` prop, taking a CSS selector as a value, so I needed to check the validity of the passed value. Then I changed my mind about the selector but I think this function could be useful so I leaved it there